src/main/java/org/jabref/model/entry/Langid.java

```diff
@@ -0,0 +1,95 @@
+package org.jabref.model.entry;
+
+import java.util.Optional;
+
+import org.jabref.logic.bibtex.FieldWriter;
+/**
+ * Language identifiers based on BibLaTeX manual specifications.
+ * See the BibLaTeX documentation for full details:
+ * <a href="http://mirrors.ctan.org/macros/latex/contrib/biblatex/doc/biblatex.pdfhangelo">BibLaTeX manual</a>
+ */
+
```

Comment:
JavaDoc needs to be attached to class --> remove empty line here (and add empty line above)

Reply:
Works with line in between